sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -285,13 +288,14 @@ def get_queue_receiver(
         :keyword session_id: A specific session from which to receive. This must be specified for a
          sessionful queue, otherwise it must be None. In order to receive messages from the next available
          session, set this to ~azure.servicebus.NEXT_AVAILABLE_SESSION.
-        :paramtype session_id: Union[str, ~azure.servicebus.NEXT_AVAILABLE_SESSION]
-        :keyword Optional[Union[ServiceBusSubQueue, str]] sub_queue: If specified, the subqueue this receiver will
+        :paramtype session_id: Union[str, Literal[~azure.servicebus.NEXT_AVAILABLE_SESSION]]
```

Comment:
I would not document it like this in the docstring. Docstring formatting != type hints.
It should be something like:
`:paramtype session_id: str or ~azure.servicebus.NEXT_AVAILABLE_SESSION`